tools/ci_build/github/azure-pipelines/linux-migraphx-ci-pipeline.yml

```diff
@@ -57,7 +57,7 @@ jobs:
               --update \
               --build_dir /build \
               --build \
-              --parallel 8 \
+              --parallel 16 \
```

Comment:
How many CPU cores the machine has?

Reply:
64 cores on this machine and we have 4 agents on it.